FireControl.cps

```diff
@@ -73,6 +74,7 @@ propertyDefinitions = {
     pierceHeight: {title:"Pierce Height (in)", description:"Height at which to pierce material.", type:"number"},
     cutHeight: {title:"Cut Height (in)", description:"Height at which to cut material.", type:"number"},
     retractHeight: {title:"Retract Height (in)", description:"Height at which to retract above material.", type:"number"},
+    pierceOnlyDetectToggle: {title:"Center pierce only detect", description:"Pierce only the center of holes for cutting operations named 'pierce-only'", type:"boolean"},
```

Comment:
Awkward title, maybe just put "Pierce Only"

Reply:
Yeah but this is specifically a hole center pierce, I wanted to convey that...